tests/test-blas-domain.C

```diff
@@ -1779,8 +1780,8 @@ int launch_tests(Field & F, size_t n, int iterations)
  	if (!testPLUQ (F,n,n,iterations))                     pass=false;
  	if (!testMinPoly (F,n,iterations))                    pass=false;
 	if (!testCharPoly (F,n,iterations))                   pass=false;
-	//
-	//
+	
+	
```

Comment:
What about some tests for left_solve and right_solve?

Reply:
I don't catch your comment. There are some usage of left_solve and right_solve in function testSolve and testTriangularSolve.